pkg/tuf/delegations.go

```diff
@@ -0,0 +1,29 @@
+package tuf
+
+import (
+	"fmt"
+
+	"github.com/theupdateframework/notary/client"
+	"github.com/theupdateframework/notary/tuf/data"
+)
+
+// Delegate all paths ("*") to targets/releases.
+// https://github.com/theupdateframework/notary/blob/f255ae779066dc28ae4aee196061e58bb38a2b49/cmd/notary/delegations.go
```

Comment:
Was this function copied in any way from TUF? I'm not sure what this link is referring to.